crypto/evp_extra/p_pqdsa_asn1.c

```diff
@@ -18,6 +18,40 @@ static void pqdsa_free(EVP_PKEY *pkey) {
   pkey->pkey.pqdsa_key = NULL;
 }
 
+static int pqdsa_get_priv_raw_seed(PQDSA_KEY *key, const PQDSA *pqdsa,
+                                  uint8_t *out,size_t *out_len) {
```

Comment:
```suggestion
static int pqdsa_get_priv_raw_seed(PQDSA_KEY *key, const PQDSA *pqdsa,
                                  uint8_t *out,0 size_t *out_len) {
GUARD_PTR(key);
GUARD_PTR(out_len);
```

Reply:
added to `PQDSA_KEY_get_priv_raw_seed` in [c307952](https://github.com/aws/aws-lc/pull/2194/commits/c30795274cf00bed09803b8f5fe6114112fb1422)